### DISCUSSION TITLE
committer: Nick Crews

### DISCUSSION THREAD

**Discussion by lostmygithubaccount** [ibis-project/governance#5] [Committer] [answered]
I would like to nominate Nick Crews (@NickCrews) for Ibis committership.

Nick has been a longtime contributor to the blog, through many issues and discussions, and even pull requests. His feedback and dedication to the project is highly valued.

**Comment by NickCrews** [comment, marked as answer]
I'm honored! I would appreciate the ability to have some more autonomy here. I would still want a meeting to get a sense of how y'all think about the review process, if there is anything that can get merged without review, etc

**Comment by kszucs** [comment]
I am in favour of the proposal. Nick has been a really active contributor for a while now, producing enormous amount of reproducible bugs, feature requests and lately PRs as well. His attitude and efforts are beneficial to the project.

**Comment by cpcloud** [comment]
+1

Wonderful to you have you around these parts @NickCrews. Look forward to continued contributions 🚀

**Comment by gforsyth** [comment]
+1 from me.
Nick has been a fantastic contributor and I look forward to his continued involvement with the project.

I _think_ (I'm on mobile) that governance states that a steering council member should nominate and then two other steering council members should approve.

A committee nominating and then 3 steering council members approving is clearly equivalent -- we may want to codify that in the governance docs.

**Reply by lostmygithubaccount** [reply]
ah my bad! gotta bend the rules to change them I suppose

**Comment by jcrist** [comment]
+1 from me! Nick has been great, look forward to working with them more in the future.

**Comment by gforsyth** [comment]
With 3 votes, the motion carries.  Welcome aboard @NickCrews !
